notebooks/book1/03/gauss_imputation_known_params_demo.ipynb

```diff
@@ -0,0 +1,322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        from probml_utils import savefig, latexify, is_latexify_enabled
</pre><p>you can add just 1 line in try...except and then other2 lines can go after that.</p><p>like this,</p><p>try:</p><p>	from probml_utils import pyprobml_utils as pml</p><p>except:</p><p>	5pip install git+https:// github.com/probml/probml-utils.git</p><p>	from probml_utils import pyprobml_utils as pml</p><p> </p><p>from probml_utils import savefig, latexify, is_latexify_enabled</p><p>from probml_utils import gauss_utils as gauss</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Oh yes, that is correct

Reply:
Thanks for catching that. Fixed this, will the master repo